modules/core/src/test/scala/gem/Arbitraries.scala

```diff
@@ -32,6 +32,9 @@ trait Arbitraries extends gem.config.Arbitraries  {
       )
     }
 
+  implicit val cogLocation: Cogen[Location] =
```

Comment:
I watched the bit about `Cogen`.  It's very nicely done.  So when I see a `Cogen[A]` now it tells me that it will allow us to create a generator of arbitrary functions from `A => B`, given a `Gen[B]` as well.  Correct?  Do we actually make use of this anywhere yet?